### ISSUE TITLE
Get all items

### ISSUE THREAD

**Issue by Pokemaster623** [hartmannaf/PokemonCheatPlugin#8]
When using get all items on Y I get a bunch of unknown items. These are probably items that exist in ORAS, but not in XY.

**Comment by hartmannaf** [comment]
No, these items exist in both version(most of them are unkown on all editions), but they  are some placeholders or something like that. For completion I included them, but if they are not somehow deletable(through selling or throwing them away), I will remove them from the list.
